app/src/main/kotlin/com/mapzen/erasermap/view/SettingsFragment.kt

```diff
@@ -118,6 +134,24 @@ public class SettingsFragment : PreferenceFragment(), Preference.OnPreferenceCha
         getSettingsActivity().clearHistory(title)
     }
 
+    private fun updateMockTracePref(mockLocationEnabled: Boolean) {
+        if (!mockLocationEnabled) {
+            val prefs = PreferenceManager.getDefaultSharedPreferences(this.context)
```

Comment:
`Fragment.getContext()` requires API 23. This will crash on older versions of Android

https://developer.android.com/reference/android/app/Fragment.html#getContext()